PR-App/PR-App/Custom Views/Cells/CustomCell.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  CustomCell.swift
+//  PR-App
+//
+//  Created by Jakub Homik on 14/05/2020.
+//  Copyright © 2020 Jakub Homik. All rights reserved.
+//
+
+import UIKit
+
+class CustomCell: UICollectionViewCell {
+    
+    var imageCell = UIImageView()
+    var newMedsTitle = UILabel()
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        configureCell()
+        configureButtonImage()
+        configureMedsTitle()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    private func configureButtonImage() {
+        imageCell.image = UIImage(systemName: "plus.circle.fill")
```

Comment:
Why are You not using Constants file? If You do not use it, it should be deleted.